modular_citadel/code/modules/reagents/reagents/cit_reagents.dm

```diff
@@ -156,7 +156,6 @@
 			var/mob/living/carbon/human/H = M
 			if(prob(50)) //Less spam
```

Comment:
lower this probability to a value of 5, merge it with line 155, placing it first on that list of checks.